### ISSUE TITLE
Bundler gem release error

### ISSUE THREAD

**Issue by remi-gelinas** [bundler/bundler#4164]
- What command did you try to run?

`bundle exec rake release`

- What did you expect to happen?

I expected Bundler to release my gem, updating the version and pushing it to GitHub and Rubygems.

- What actually happened?

Bundler crashed immediately with this stack trace:

```
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/x86_64-darwin15/psych.bundle: warning: already initialized constant Psych::Parser::ANY
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/x86_64-darwin15/psych.bundle: warning: already initialized constant Psych::Parser::UTF8
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/x86_64-darwin15/psych.bundle: warning: already initialized constant Psych::Parser::UTF16LE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/x86_64-darwin15/psych.bundle: warning: already initialized constant Psych::Parser::UTF16BE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:6: warning: already initialized constant Psych::ClassLoader::BIG_DECIMAL
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:6: warning: previous definition of BIG_DECIMAL was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:7: warning: already initialized constant Psych::ClassLoader::COMPLEX
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:7: warning: previous definition of COMPLEX was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:8: warning: already initialized constant Psych::ClassLoader::DATE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:8: warning: previous definition of DATE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:9: warning: already initialized constant Psych::ClassLoader::DATE_TIME
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:9: warning: previous definition of DATE_TIME was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:10: warning: already initialized constant Psych::ClassLoader::EXCEPTION
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:10: warning: previous definition of EXCEPTION was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:11: warning: already initialized constant Psych::ClassLoader::OBJECT
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:11: warning: previous definition of OBJECT was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:12: warning: already initialized constant Psych::ClassLoader::PSYCH_OMAP
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:12: warning: previous definition of PSYCH_OMAP was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:13: warning: already initialized constant Psych::ClassLoader::PSYCH_SET
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:13: warning: previous definition of PSYCH_SET was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:14: warning: already initialized constant Psych::ClassLoader::RANGE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:14: warning: previous definition of RANGE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:15: warning: already initialized constant Psych::ClassLoader::RATIONAL
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:15: warning: previous definition of RATIONAL was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:16: warning: already initialized constant Psych::ClassLoader::REGEXP
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:16: warning: previous definition of REGEXP was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:17: warning: already initialized constant Psych::ClassLoader::STRUCT
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:17: warning: previous definition of STRUCT was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:18: warning: already initialized constant Psych::ClassLoader::SYMBOL
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:18: warning: previous definition of SYMBOL was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/class_loader.rb:64: warning: already initialized constant Psych::ClassLoader::CACHE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/class_loader.rb:64: warning: previous definition of CACHE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/scalar_scanner.rb:8: warning: already initialized constant Psych::ScalarScanner::TIME
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/scalar_scanner.rb:8: warning: previous definition of TIME was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/scalar_scanner.rb:11: warning: already initialized constant Psych::ScalarScanner::FLOAT
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/scalar_scanner.rb:11: warning: previous definition of FLOAT was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/scalar_scanner.rb:17: warning: already initialized constant Psych::ScalarScanner::INTEGER
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/scalar_scanner.rb:17: warning: previous definition of INTEGER was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/stream.rb:12: warning: already initialized constant Psych::Nodes::Stream::ANY
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/stream.rb:12: warning: previous definition of ANY was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/stream.rb:15: warning: already initialized constant Psych::Nodes::Stream::UTF8
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/stream.rb:15: warning: previous definition of UTF8 was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/stream.rb:18: warning: already initialized constant Psych::Nodes::Stream::UTF16LE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/stream.rb:18: warning: previous definition of UTF16LE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/stream.rb:21: warning: already initialized constant Psych::Nodes::Stream::UTF16BE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/stream.rb:21: warning: previous definition of UTF16BE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/sequence.rb:42: warning: already initialized constant Psych::Nodes::Sequence::ANY
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/sequence.rb:42: warning: previous definition of ANY was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/sequence.rb:45: warning: already initialized constant Psych::Nodes::Sequence::BLOCK
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/sequence.rb:45: warning: previous definition of BLOCK was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/sequence.rb:48: warning: already initialized constant Psych::Nodes::Sequence::FLOW
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/sequence.rb:48: warning: previous definition of FLOW was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/scalar.rb:9: warning: already initialized constant Psych::Nodes::Scalar::ANY
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/scalar.rb:9: warning: previous definition of ANY was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/scalar.rb:12: warning: already initialized constant Psych::Nodes::Scalar::PLAIN
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/scalar.rb:12: warning: previous definition of PLAIN was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/scalar.rb:15: warning: already initialized constant Psych::Nodes::Scalar::SINGLE_QUOTED
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/scalar.rb:15: warning: previous definition of SINGLE_QUOTED was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/scalar.rb:18: warning: already initialized constant Psych::Nodes::Scalar::DOUBLE_QUOTED
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/scalar.rb:18: warning: previous definition of DOUBLE_QUOTED was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/scalar.rb:21: warning: already initialized constant Psych::Nodes::Scalar::LITERAL
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/scalar.rb:21: warning: previous definition of LITERAL was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/scalar.rb:24: warning: already initialized constant Psych::Nodes::Scalar::FOLDED
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/scalar.rb:24: warning: previous definition of FOLDED was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/mapping.rb:16: warning: already initialized constant Psych::Nodes::Mapping::ANY
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/mapping.rb:16: warning: previous definition of ANY was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/mapping.rb:19: warning: already initialized constant Psych::Nodes::Mapping::BLOCK
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/mapping.rb:19: warning: previous definition of BLOCK was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/nodes/mapping.rb:22: warning: already initialized constant Psych::Nodes::Mapping::FLOW
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/nodes/mapping.rb:22: warning: previous definition of FLOW was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/visitors/visitor.rb:10: warning: already initialized constant Psych::Visitors::Visitor::DISPATCH
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/visitors/visitor.rb:10: warning: previous definition of DISPATCH was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/handler.rb:26: warning: already initialized constant Psych::Handler::OPTIONS
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/handler.rb:26: warning: previous definition of OPTIONS was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/handler.rb:29: warning: already initialized constant Psych::Handler::EVENTS
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/handler.rb:29: warning: previous definition of EVENTS was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/visitors/yaml_tree.rb:430: warning: already initialized constant Psych::Visitors::YAMLTree::NULL
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/visitors/yaml_tree.rb:439: warning: previous definition of NULL was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/visitors/yaml_tree.rb:431: warning: already initialized constant Psych::Visitors::YAMLTree::BINARY_RANGE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/visitors/yaml_tree.rb:440: warning: previous definition of BINARY_RANGE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/visitors/yaml_tree.rb:432: warning: already initialized constant Psych::Visitors::YAMLTree::WS_RANGE
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/psych-2.0.16/lib/psych/visitors/yaml_tree.rb:441: warning: previous definition of WS_RANGE was here
/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/parser.rb:33:in `<class:Parser>': superclass mismatch for class Mark (TypeError)
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/parser.rb:32:in `<module:Psych>'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych/parser.rb:1:in `<top (required)>'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych.rb:7:in `require'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/2.2.0/psych.rb:7:in `<top (required)>'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems.rb:612:in `require'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems.rb:612:in `load_yaml'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/config_file.rb:325:in `load_file'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/config_file.rb:197:in `initialize'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/gem_runner.rb:74:in `new'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/gem_runner.rb:74:in `do_configuration'
	from /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/gem_runner.rb:39:in `run'
	from /Users/dummy/.rbenv/versions/2.2.3/bin/gem:21:in `<main>'

Tasks: TOP => release => build
(See full trace by running task with --trace)
```

- What is your `bundler env`?

```
Environment

    Bundler   1.11.0
    Rubygems  2.5.1
    Ruby      2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin15]
    GEM_HOME  /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0
    GEM_PATH  /Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0:/Users/dummy/.gem/ruby/2.2.0
    Git       2.6.4

Bundler settings

    gem.coc
      Set for the current user (/Users/dummy/.bundle/config): "false"
    gem.mit
      Set for the current user (/Users/dummy/.bundle/config): "false"
    gem.test
      Set for the current user (/Users/dummy/.bundle/config): "rspec"
    bin_path
      Set via BUNDLE_BIN_PATH: "/Users/dummy/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.0/exe/bundle"
    gemfile
      Set via BUNDLE_GEMFILE: "/Users/dummy/Workspace/dummy/Gemfile"

Gemfile

    source 'https://rubygems.org'

    # Specify your gem's dependencies in dummy.gemspec
    gemspec

Gemfile.lock

    PATH
      remote: .
      specs:
        dummy (1.1.0)

    GEM
      remote: https://rubygems.org/
      specs:
        diff-lcs (1.2.5)
        rake (10.4.2)
        rspec (3.4.0)
          rspec-core (~> 3.4.0)
          rspec-expectations (~> 3.4.0)
          rspec-mocks (~> 3.4.0)
        rspec-core (3.4.1)
          rspec-support (~> 3.4.0)
        rspec-expectations (3.4.0)
          diff-lcs (>= 1.2.0, < 2.0)
          rspec-support (~> 3.4.0)
        rspec-mocks (3.4.0)
          diff-lcs (>= 1.2.0, < 2.0)
          rspec-support (~> 3.4.0)
        rspec-support (3.4.1)

    PLATFORMS
      ruby

    DEPENDENCIES
      bundler (~> 1.10)
      dummy!
      rake (~> 10.0)
      rspec

    BUNDLED WITH
       1.11.0

dummy.gemspec:

    # coding: utf-8
    lib = File.expand_path('../lib', __FILE__)
    $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
    require 'dummy/version'

    Gem::Specification.new do |spec|
      spec.name          = "dummy"
      spec.version       = Dummy::VERSION
      spec.authors       = ["dummy"]
      spec.email         = ["dummy"]

      spec.summary       = %q{dummy}
      spec.description   = %q{dummy}
      spec.homepage      = "dummy"

      spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
      spec.bindir        = "exe"
      spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
      spec.require_paths = ["lib"]

      spec.add_development_dependency "bundler", "~> 1.10"
      spec.add_development_dependency "rake", "~> 10.0"
      spec.add_development_dependency "rspec"
    end
```

**Comment by rrooding** [comment]
It looks like this was fixed in 1.11.1 which was just released. See #4149

**Comment by mupkoo** [comment]
I have the same problem using `1.11.1` and trying to update `rails` via `bundle update rails`

Error details

    RuntimeError: LazySpecification has not been materialized yet (calling :required_ruby_version [])
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/lazy_specification.rb:75:in `method_missing'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:132:in `for?'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:275:in `block in search_for'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:275:in `select'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:275:in `search_for'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:332:in `block in verify_gemfile_dependencies_are_found!'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:330:in `each'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:330:in `verify_gemfile_dependencies_are_found!'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:199:in `start'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/resolver.rb:183:in `resolve'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/definition.rb:198:in `resolve'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/definition.rb:137:in `specs'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/definition.rb:126:in `resolve_remotely!'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/installer.rb:194:in `resolve_if_need'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/installer.rb:69:in `run'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/installer.rb:21:in `install'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/cli/update.rb:51:in `run'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/cli.rb:197:in `update'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/cli.rb:10:in `start'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/exe/bundle:19:in `block in <top (required)>'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/lib/bundler/friendly_errors.rb:7:in `with_friendly_errors'
      /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.11.1/exe/bundle:17:in `<top (required)>'
      /Users/Mupko/.rbenv/versions/2.2.3/bin/bundle:23:in `load'
      /Users/Mupko/.rbenv/versions/2.2.3/bin/bundle:23:in `<main>'

Environment

    Bundler   1.11.1
    Rubygems  2.5.1
    Ruby      2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin15]
    GEM_HOME  /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0
    GEM_PATH  /Users/Mupko/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0:/Users/Mupko/.gem/ruby/2.2.0
    Git       2.6.4
